### ISSUE TITLE
Download directly from default site (category)

### ISSUE THREAD

**Issue by strymsg** [strymsg/almacenArchivos#3]

**Comment by strymsg** [comment]
When trying to download a file using a now browser agent like wget, returns 308 not the file, for instance:
```bash
$ wget http://localhost:5000/almacen/Misc/COPYING
--2020-02-04 17:20:58--  http://localhost:5000/almacen/Misc/COPYING
Resolviendo localhost (localhost)... 127.0.0.1
Conectando con localhost (localhost)[127.0.0.1]:5000... conectado.
Petición HTTP enviada, esperando respuesta... 308 PERMANENT REDIRECT
2020-02-04 17:20:58 ERROR 308: PERMANENT REDIRECT.

```

**Comment by strymsg** [comment]
Fixed on branch #hotfix#3
